packages/admin/dashboard/scripts/i18n/validate-translation.js

```
@@ -1,47 +1,135 @@
 const Ajv = require("ajv")
 const fs = require("fs")
 const path = require("path")
+
 const schema = require("../../src/i18n/translations/$schema.json")
+const pluralConfig = require("../../src/i18n/plural-config.json")
 
-const ajv = new Ajv({ allErrors: true })
-const validate = ajv.compile(schema)
+const ajv = new Ajv({ allErrors: true, strict: false })
+const validateSchema = ajv.compile(schema)
 
-// Get file name from command line arguments
 const fileName = process.argv[2]
-
 if (!fileName) {
   console.error("Please provide a file name (e.g., en.json) as an argument.")
   process.exit(1)
 }
 
+const langCode = fileName.replace(".json", "")
+const requiredForms = pluralConfig[langCode]
+
+if (!requiredForms) {
+  console.error(`Language "${langCode}" not found in plural-config.json`)
```

Maybe there should be a default? Most of the languages use what kind of system?

---

Most languages use the same plural system as English (`one`, `other`). I intentionally didn’t include a default, though, because I want to ensure that whenever someone adds a new language, they also explicitly define its plural rules in `plural-config.json`.

This prevents silent errors for example, adding a language that appears to work, but actually requires more plural forms.

The idea is that after this PR is merged, we’ll add a note to the documentation explaining that every new locale must include its plural configuration. That way we keep consistency and avoid breaking pluralization unintentionally.